### PR TITLE
Added NSFW Commands

### PR DESCRIPTION
**Pull request by GrandMoff100**

### CODE REVIEW

**Comment by GrandMoff100** [comment]
Doesn't contain docstrings.

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:12]:
```suggestion
    embed = discord.Embed(color=discord.Colour.red())
    embed.title = provider + " requested by " + author.name
    embed.set_image(url=img)
    return embed
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:27]:
```suggestion
        req = requests.get(base + url)
        req = req.json()
        embed = await make_embed(req['data']['response']["url"], "Nekos.life", author)
        return embed
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:43]:
```suggestion
            embed = await self.get(source, ctx.author)
            msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:44]:
```suggestion
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:54]:
```suggestion
        embed = await self.get(source, ctx.author)
        msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:63]:
```suggestion
            embed = await self.get(source, ctx.author)
            msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:72]:
```suggestion
    @neko.command(name="kitsune")
    @commands.guild_only()
    async def sfw_kitsune(self, ctx):
        async with ctx.typing():
            source = "images/sfw/img/kitsune"
            embed = await self.get(source, ctx.author)
            msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:82]:
```suggestion
        embed = await self.get(source, ctx.author)
        msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:92]:
```suggestion
        embed = await self.get(source, ctx.author)
        msg = await ctx.send(embed=embed)
```

File: bot/cogs/nsfw/neko.py
Comment (janaSunrise) [inline bot/cogs/nsfw/neko.py:102]:
```suggestion
        embed = await self.get(source, ctx.author)
        msg = await ctx.send(embed=embed)
```

**Review by janaSunrise — CHANGES_REQUESTED** [review]
Please request those changes, and edit the files to use proper naming convention, and try to make it different, so it clearer

**Review by ItsDrike — CHANGES_REQUESTED** [review]
While I've requested some changes to code itself isn't that bad, although I do want to state my position on this which is that I'm personally not a huge fan of having NSFW commands like this in the bot and after these changes will get resolved I won't approve this PR because of that if it will get 2 approvals anyway, it can be merged even though I'm personally against having this in our bot.

File: bot/cogs/reddit.py
Comment (janaSunrise) [reply bot/cogs/reddit.py:65]: i got this error from flake8-bear

Comment (ItsDrike) [reply bot/cogs/reddit.py:65]: Can you show me the specific error number with corresponding message to it?

**Pull request merged**
